rpcserver.go

```diff
@@ -1832,6 +1852,10 @@ func (r *rpcServer) SendPayment(paymentStream lnrpc.Lightning_SendPaymentServer)
 					p.cltvDelta = uint16(nextPayment.FinalCltvDelta)
 				}
 
+				p.feeLimit = calculateFeeLimit(
```

Comment:
Make an addition to the integration tests to check that percentage and absolute fee limits are working correctly?

Reply:
Fixed.